web/libs/datamanager/src/components/Common/Table/TableHead/TableHead.jsx

```diff
@@ -218,7 +218,10 @@ export const TableHead = observer(
 
         useEffect(() => {
           ref.current?.addEventListener("mousedown", (event) => {
-            if (event.target.className.includes("handle")) event.preventDefault();
+            const className = event.target.className;
+            if (typeof className === "string" && className.includes("handle")) {
```

Comment:
This would also work, as in the case of className being anything other than a string the method includes will also be undefined.
```suggestion
            if (className?.includes?.("handle")) {
```

Reply:
Do you feel strong about this? I will go either way.
I'd like to show that I'm using "includes" because it's a string. But, again,  if you feel strong about it I will change it.

Reply:
I would also vote for null check, because it's more semantically correct. In your code it feels like `className` can be some other type which is not the case, right? We are just protecting us against the target without class name (what the case is it btw?)

Reply:
ah, sorry, I re-read the description and now see that it can actually be a different kind of "string". Then both solutions are good, but they both require a comment. (still vote for shorter version though)

Reply:
Update to use the shorter version since both of you agree :)

Reply:
Also added a comment